### PR TITLE
Update machine-translation-and-dataset.md 

### PR DESCRIPTION
**Pull request by tcmyxc**
修改不合规范的标点符号

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-1008/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-1008/

File: chapter_attention-mechanisms/nadaraya-waston.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ n_test
 def plot_kernel_reg(y_hat):
     d2l.plot(x_test, [y_truth, y_hat], 'x', 'y', legend=['Truth', 'Pred'],
              xlim=[0, 5], ylim=[-1, 5])
-    d2l.plt.plot(x_train, y_train, 'o', alpha=0.5);
+    d2l.plt.plot(x_train, y_train, 'o', alpha=0.5)
```
Comment (goldmermaid) [inline chapter_attention-mechanisms/nadaraya-waston.md:96]: we may still need this `;`, or the plot may not render... can you add it back?

Comment (tcmyxc) [reply chapter_attention-mechanisms/nadaraya-waston.md]: OK

**Comment by mli** [comment]
Job d2l-zh/PR-1008/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-1008/

**Comment by mli** [comment]
Job d2l-zh/PR-1008/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-1008/

**Comment by goldmermaid** [comment]
Thanks @tcmyxc !

**Pull request merged**